tests/test_line.py

```diff
@@ -272,3 +272,32 @@ def test_from_dict_current():
     assert d1.length == 4
 
     assert d2 is d1
+
+
+def test_optimize_multipoles():
```

Comment:
I would rename FastQuadrupole in to **SimpleThinQuadrupole** and FastDipole into **SimpleThinBend**.
Of course all names in tests, examples, etc. need to follow